data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/RestApiConstants.java

```diff
@@ -18,6 +18,10 @@
   public static final String SLOT = "slot";
   public static final String EPOCH = "epoch";
   public static final String ACTIVE = "active";
+  public static final String PAGE_SIZE = "pageSize";
+  public static final String PAGE_TOKEN = "pageToken";
+  public static final int PAGE_SIZE_DEFAULT = 20;
```

Comment:
prysm defaults this to 250, i think we should do the same. even if we went as high as 500, just want to make sure that we get that middle ground of 'not too much' data.

Reply:
done